frontend/src/i18n/locales/de/notifications.json

```diff
@@ -0,0 +1,14 @@
+{
+  "You don't have any notifications right now": "Sie haben im Moment keine Benachrichtigungen",
+  "No message": "Keine Nachricht",
+  "Mark as read": "Als gelesen markieren",
+  "Show notifications": "Benachrichtigungen anzeigen",
+  "You have unread notifications": "Sie haben ungelesene Benachrichtigungen",
```

Comment:
Sie haben ungelesene Meldungen

Reply:
I changed it to "Sie haben ungelesene Meldungen". But seeing separate correction here... I'm wondering if it should be "Sie haben ungelesene Nachrichten".